exonum-java-binding/CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `supervisor-mode` CLI parameter added for `generate-template` command. It
   allows to configure the mode of the Supervisor service. Possible values are
   "simple" and "decentralized". (#1361)
-- Service instances can be stopped now. (#1358)
+- Service instances can be stopped and resumed now. (#1358, #1372)
```

Comment:
An instance artifact may be upgraded before resuming, but with no data changes. 

(may also insert the uses cases from the epic/ECR-3773).